docs/manuals/man5/pki-tps-profile.5.md

```diff
@@ -17,7 +17,7 @@ Token profiles are defined using properties in the TPS configuration file.
 The following property sets the size of the key the token should generate:
 
 ```
-op.enroll.<tokenType>.keyGen.<keyType>.keySize=1024
+op.enroll.<tokenType>.keyGen.<keyType>.keySize=2048
 ```
 
 The maximum value is 1024.
```

Comment:
I'm not familiar with this, but if the maximum value is referring to the `keySize` parameter, would they be contradictory now.

Reply:
Maybe it depends on the key type? DSA has a strict 1024-bit maximum, ECDSA obviously differs based on curve type/size and isn't user controllable, and for RSA, most implementations top out around 8192-bit ish.

Reply:
> The valid algorithms are:
> - **2** - RSA
> - **5** - ECC
> For ECC, the valid key sizes are 256 and 384.

Reply:
Or maybe this line should just be removed, and let the actual maximum be determined by the crypto library.